ltr_scripts/tf_ranking_libsvm.py

```diff
@@ -356,6 +356,7 @@ def get_eval_metric_fns():
   metric_fns.update({
       "metric/%s" % name: tfr.metrics.make_ranking_metric_fn(name) for name in [
           tfr.metrics.RankingMetricKey.ARP,
+          tfr.metrics.RankingMetricKey.MRR, # Mean Reciprocal Rank
```

Comment:
Where is this used?

Reply:
This reports mean reciprocal rank as the model is trained, so we can visualise the effect of training in tensorboard (it doesn't impact training as far as I am aware, it's just logging metrics):

<img width="1440" alt="Screenshot 2019-12-02 at 16 04 59" src="https://user-images.githubusercontent.com/8124374/69974617-86eaef80-151d-11ea-94bf-af7cd305996a.png">